test/e2e/tests/swap-send/swap-send-eth.spec.ts

```diff
@@ -74,12 +74,6 @@ describe('Swap-Send ETH', function () {
           // TODO assert swap api request payload
 
           await swapSendPage.submitSwap();
-          await swapSendPage.verifyHistoryEntry(
-            'Send ETH as TST',
-            'Pending',
-            '-1 ETH',
-            '',
-          );
           await swapSendPage.verifyHistoryEntry(
             'Send ETH as TST',
             'Confirmed',
```

Comment:
we are asserting that the expected tx is there and confirmed, which is what we really want. If we wanted to test transaction states (from unapproved, to pending to confirmed) this would need to be done in a separate test, and have a different approach (lowering the gas, to make sure that the pending tx stays there until speed up/cancel, for example)